ValveKeyValue/ValveKeyValue.Test/Text/StreamsTestCase.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Globalization;
+using NUnit.Framework;
+using ValveKeyValue.Test.Helpers;
+
+namespace ValveKeyValue.Test;
+
+internal class StreamsTestCase
+{
+    [TestCase(1)]
+    [TestCase(4096)]
```

Comment:
I don't think `4096` gets us as blocking stream, as per the previous comment. The default buffer size is `1024`.